detox/src/devices/Device.js

```diff
@@ -49,7 +50,7 @@ class Device {
     }
 
     const baseLaunchArgs = {
-      ...this._deviceConfig.launchArgs,
+      ...this._launchArgs.get(),
```

Comment:
@d4vidi , I am confused — so how I am supposed to suppress base launchArgs if passing `null` or `undefined` simply removes properties inside `this._launchArgs`?

Reply:
Why do we need an extra complexity here? Can't we just have it as a blank key-value map and merge at the very last stage all these three: `this._deviceConfig.launchArgs, this._launchArgs.get(), ...params.launchArgs`?

Is the goal to make sure users can dynamically read the launch args? Do we really need this much flexibility?

Reply:
Okay, now I see. You want to be able to reset all launchArgs forcibly. Makes sense.